java/test/jmri/jmrit/display/layoutEditor/LayoutEditorToolsTest.java

```diff
@@ -28,10 +28,18 @@ public void testCtor() {
     @Before
     public void setUp() throws Exception {
         JUnitUtil.setUp();
+        apps.tests.Log4JFixture.setUp();
+        // reset the instance manager.
+        JUnitUtil.resetInstanceManager();
```

Comment:
I missed this on my last time through.  This file also needs setUp and tearDown restored.

Reply:
Done!